client/luigi-element.d.ts

```diff
@@ -414,4 +414,27 @@ export interface LuigiClient {
   linkManager: () => LinkManager;
   uxManager: () => UxManager;
   publishEvent: (event: Event) => void;
+  /**
+   * Sets node parameters in Luigi Core. The parameters will be added to the URL.
+   * @param {Object} params
+   * @param {boolean} keepBrowserHistory
+   * @memberof LuigiClient
+   */
+  addNodeParams: (params: Object, keepBrowserHistory: boolean) => void;
+  /**
+   * Returns the node parameters of the active URL.
+   * Node parameters are defined like URL query parameters but with a specific prefix allowing Luigi to pass them to the micro frontend view. The default prefix is **~** and you can use it in the following way: `https://my.luigi.app/home/products?~sort=asc&~page=3`.
+   * <!-- add-attribute:class:warning -->
+   * > **NOTE:** some special characters (`<`, `>`, `"`, `'`, `/`) in node parameters are HTML-encoded.
+   * @param {boolean} shouldDesanitise defines whether the specially encoded characters should be desanitised
+   * @returns {Object} node parameters, where the object property name is the node parameter name without the prefix, and its value is the value of the node parameter. For example `{sort: 'asc', page: 3}`
+   * @memberof LuigiClient
+   */
+  getNodeParams: (shouldDesanitise: boolean) => Object;
+  /**
+   * Sends anchor to Luigi Core. The anchor will be added to the URL.
+   * @param {string} anchor
+   * @memberof LuigiClient
+   */
+  setAnchor: (anchor: string) => void;
```

Comment:
I tried to test this function on simpledev but I get a weird behavior. 

Luigi config: 
```
      {
          label: 'Modal WC',
          pathSegment: 'modalwc',
          hideFromNav: false,
          loadingIndicator: {
            enabled: false
          },
          context: {
            test: 12333
          },
          openNodeInModal: true,
          hideSideNav: false,
          webcomponent: true,
          viewUrl: '/helloWorldWC.js'
        }
```


WC: helloWorld.js
```
import { LuigiElement } from "./luigi-element.js";
export default class extends LuigiElement {
  render(ctx) {
    return '<div>test</div>';
  }
  onContextUpdate(ctx) {
    console.log('context update', ctx);
  }

  afterInit(ctx) {
       this.LuigiClient.setAnchor('A')
  }
}
```
**Prerequisite**: the main microfrontend opened should be iframe based and Anchor shouldn't already be set. 

When I try to open the **Modal WC**, with the **setAnchor** being run. The modal quickly opens and then closes directly after.  If I open modal second time then it works. I guess should be checked.

Reply:
Good catch. It's related to a unexpected behavior that already exists. It's discussed with Philipp that we handle this in another issue. It's just not allowed in this issue to call setAnchor in a modal.  Also because calling setAnchor in a modal is not a normal use case.